cfme/services/requests.py

```diff
@@ -197,6 +197,30 @@ def debug_requests():
             logger.debug(' {}'.format(row))
 
 
+def go_to_request_page(cells):
+    """Finds the request and opens the page
```

Comment:
psav , I just want to go to request page here and see if the row exist. Don't want to click on the row.